Application/Features/WorkplaceFeature/Queries/GetWorkplaceListQuery.cs

```diff
@@ -35,6 +37,14 @@ public async Task<GetWorkplaceListQueryResponse> Handle(GetWorkplaceListQueryReq
             {
                 workplaces = workplaces.Where(x => x.MapId == Int32.Parse(query.MapId));
             }
+            if (query.Booking is not null)
+            {
+                //finding free workplaces Id in this period
+                var freeWorkplacesInThisPeriod= _context.Bookings.Where(x =>                            
+                          (query.Booking.EndDate < x.StartDate) || (x.EndDate < query.Booking.StartDate)
```

Comment:
Can the endDate be equal the startDate?

Reply:
technically yes, for 1 day time off

Reply:
yes, if booking is for 1 day in that case startDate and endDate will be equal